packages/liveblocks-node/src/__tests__/client.test.ts

```diff
@@ -157,7 +198,178 @@ describe("client", () => {
     });
   });
 
-  test("should throw an ApiError when getRoom receives an error response", async () => {
+  test("should return the edited comment when editComment receives a successful response", async () => {
+    const commentData = {
+      body: {
+        version: 1 as const,
+        content: [],
+      },
+      editedAt: new Date(),
+    };
+
+    server.use(
+      http.post(
+        `${DEFAULT_BASE_URL}/v2/rooms/:roomId/threads/:threadId/comments/:commentId`,
+        async ({ request }) => {
+          const data = await request.json();
+
+          if (JSON.stringify(data) === JSON.stringify(commentData)) {
+            return HttpResponse.json(comment);
+          }
+
+          return HttpResponse.error();
+        }
+      )
+    );
+
+    const client = new Liveblocks({ secret: "sk_xxx" });
+    const res = await client.editComment({
```

Comment:
I've not been part of this discussion, so maybe this has already been discussed at length and decided. Please feel free to discard this comment in that case!

I noticed the verb "edit" here. In an API, I'd personally prefer to use the more common verb "update". To me "edit" is more of a UI verb, e.g. you put the UI into edit mode, and then when you submit the changes, you _update_ the resource by making the API call.

(Again, feel free to disregard if this was already discussed/decided on.)

Reply:
We discussed this briefly in our check in yesterday with @marcbouchenoire , @FlowFlorent and myself. The reason we opted for the verb `edit` for the node methods is mostly to keep it consistent with our client hooks/functions (`useEditThreadMetadata`, `useEditComment`).